### PR TITLE
From Address Publish mechanism

### PR DESCRIPTION
**Pull request by njseeto**
[Trello](https://trello.com/c/eG608how/2928-publishing-add-send-from-address)

### Remove service email from from email updater
We no longer need the `SERVICE_EMAIL_FROM` service config here, as we will move this into the `PublishServiceCreation` class.
We need to separate the `SERVICE_EMAIL_FROM` out of the email settings page to be able to surface up to three different emails (ie: if the user has different emails published to Test, Live and saved in the `FromAddress` database) on the email settings page.

### Add service configuration for service email from
Now that we have removed the `SERVICE_EMAIL_FROM` from the `EmailUpdater`, we need to create the `SERVICE_EMAIL_FROM` config in the `PublishServiceCreation`. The `PublishServiceCreation` is only ever instantiated when a user clicks publish for a given service.
If the `SERVICE_EMAIL_FROM` config does not exist for a service, we create one with the existing email in the FromAddress record for that environment.

### Add the manual gate back into the deployment pipeline
This is because we are changing the Publishing mechanism, which we would like to test fully before deploying to Live.
Once we are happy with the behaviours in testing, we can move back to continuous deployment to Live.

### CODE REVIEW

File: app/services/publish_service_creation.rb [hunk 1]
```diff
@@ -163,4 +170,16 @@ def service_csv_output
       deployment_environment: deployment_environment
     ).try(:service_csv_output?)
   end
+
+  def service_email_from
```
Comment (brenetic) [inline app/services/publish_service_creation.rb:174]: where is this being used?

File: app/services/publish_service_creation.rb [hunk 2]
```diff
@@ -75,6 +76,12 @@ def no_service_output?
     send_by_email.blank? || (send_by_email.present? && service_email_output.blank?)
   end
 
+  def assign_service_email_from
```
Comment (brenetic) [inline app/services/publish_service_creation.rb:79]: can probably move it to be private

File: app/services/publish_service_creation.rb (hunk 1 above)
Comment (njseeto) [reply app/services/publish_service_creation.rb]: good spot, forgot to remove in the clean up.

**Review by brenetic — APPROVED** [review]
🚀

**Pull request merged**